input/_ExtensionsList.cshtml

```diff
@@ -0,0 +1,83 @@
+@model IEnumerable<IDocument>
+
+<div class="alert alert-info">
+    <p>
+        This page contains extensions for Cake.
+        If you want to file an issue for any extension listed here, contact the author of the addin.
```

Comment:
You mention addin wrote the end of this line, is that intended?

Reply:
Nope, chaned to "extension" (at the moment it still only shows addins though)